packages/module/patternfly-docs/content/design-guidelines/design-guidelines.md

```diff
@@ -9,6 +9,22 @@ id: User feedback
 source: design-guidelines
 ---
 
-Design guidelines
+Note: User feedback lives in its own package at [@patternfly/react-user-feedback](https://github.com/patternfly/react-user-feedback).
 
-COMING SOON!
+**User feedback** is a modal component that on-premises products can use to collect asynchronous feedback from users. 
+
+## Behavior
+
+When installed, user feedback may be accessed in a couple of ways: 
+
+By selecting a masthead’s help icon and choosing “Share feedback” in the dropdown menu:
+
+![Image of expanded dropdown menu with link to sharing feedback](/packages/module/patternfly-docs/content/img/dropdown-selection.png)
+
+By selecting “Share feedback” links or buttons elsewhere in the console, such as at the end of a wizard.
+
+![Image of feedback options in a wizard](/packages/module/patternfly-docs/content/img/wizard-selection.png)
+
+By default, user feedback can point users to 2 Qualtrics forms: a [feedback form](https://console.redhat.com/self-managed-feedback-form), which asks users to offer feedback on their product experience, and a [research form]( https://console.redhat.com/self-managed-research-form), which asks users if they’d like to participate in future research with Red Hat’s User Experience Design team.
```

Comment:
We should probably change this to describing how they have the option to use URLs, or make network calls.   The user has to provides these URLs for a form they want to access.  The examples where just using qualtrics form. 

I will open an issue to change this to some other form that's not related to redhat.